cmd/kube-controller-manager/app/options/options.go

```diff
@@ -243,6 +248,7 @@ func (s *KubeControllerManagerOptions) Flags(allControllers []string, disabledBy
 	fs := fss.FlagSet("misc")
 	fs.StringVar(&s.Master, "master", s.Master, "The address of the Kubernetes API server (overrides any value in kubeconfig).")
 	fs.StringVar(&s.Kubeconfig, "kubeconfig", s.Kubeconfig, "Path to kubeconfig files with authorization and master location information.")
+	fs.StringVar(&s.ResourceProviderKubeConfig, "resource-providers", s.ResourceProviderKubeConfig, "Path to kubeconfig files points to resource provider(s).")
```

Comment:
does it accept multiple rps? if so, is the string comma separated?

Reply:
Yes it accepts multiple rps and filenames are comma separated